### ISSUE TITLE
When selecting a recording that has no coverart, the previous coverart is displayed.

### ISSUE THREAD

**Issue by billmeek** [MythTV-Clients/MythTV-Android-Frontend#68]
I've been taking the latest Recordings screen out for a test drive, and understand it to be work-in-progress.

When I select a recording that has no coverart, the coverart displayed from the previous recording remains. All of the text displayed is correct.

There are two short logs here: http://paste.ubuntu.com/1313523/ along with the steps used to produce the condition.

The addition of coverart and the ability to select each recording to see its detail looks very good. I like that I can do that and then choose to view it by pressing the eyeball!

**Comment by dmfrey** [comment]
Do a pull and try it again. This should be fixed as of my last commit.

**Comment by billmeek** [comment]
Hey Dan;

Now I get no recordings at all, http://paste.ubuntu.com/1313681/ near the bottom, I can see the 'deleted artwork directory' messages. Looking with the 'My Files' app, the 40 directories under org.mythtv cache cache programs are empty.

No worries, I'll just be patient.

Bill

**Comment by dmfrey** [comment]
Try hitting the refresh button. Do you get a toast message saying the
recorded programs were updated?

**Comment by billmeek** [comment]
Good point, I did refresh, but didn't look at the message text. It says:

<pre>
Recorded Programs are up to date!
</pre>

**Comment by dmfrey** [comment]
And still no program groups or episodes?

**Comment by billmeek** [comment]
Correct. I haven't pulled a new log, but looking at line 590 of the last one, I see:

<pre>
10-28 16:49:48.040: I/RecordedDownloadService(2137): download : GetRecordedList returned 304 Not Modified.
</pre>

Belay that, I just pressed refresh four more times, waiting for a response after each press and the programs are back.

Guessing here, at 6:00 (which is now, my time) the program 60 Minutes starts recording and that added an entry to the recordings page. Perhaps that's why I've got a full set of recordings.

Also, my original comment about 'coverart-less' recordings getting the last select coverart is fixed, as you suggested.

Since the subject of this issue is fixed, I'll close it. If the 'no recordings case' can be duplicated, I'll open a new issue (unless you've got enough to go on based on the above.)
